data/lib/compat/compat.lua

```diff
@@ -1006,3 +1006,21 @@ end
 function Guild.removeMember(self, player)
 	return player:getGuild() == self and player:setGuild(nil)
 end
+
+function doSetItemOutfit(cid, item, time) local c = Creature(cid) return c ~= nil and c:setItemOutfit(item, time) end
+function doSetMonsterOutfit(cid) local c = Creature(cid) return c ~= nil and c:setMonsterOutfit(name, time) end
```

Comment:
Correct haha, just copied your comment and only tested the doSetCreatureOutfit compat function :P